src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs

```diff
@@ -64,4 +64,4 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyFileVersion("2.0.2.6067")]
+[assembly: AssemblyFileVersion("2.0.2.6426")]
```

Comment:
These changes don't need to be committed and can be excluded

Reply:
i'm not doing anything on purpose here (include or exclude) - do you mean I should manually exclude (not commit) them ? Feels like there should be something in `.gitignore` for it ?